torch/csrc/jit/ir.cpp

```diff
@@ -251,6 +256,357 @@ std::ostream& operator<<(std::ostream & out, const Graph & g) {
   return out;
 }
 
+class PrettyPrintPass {
```

Comment:
Can we move this to a different file? It doesn't need to live in `ir.cpp` where it increases compilation time, and doesn't need to be here.